### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jeeva11111**
added

### CODE REVIEW

**Comment by chrisdavidmills** [comment]
Please don't use MDN as a testing sandbox. A lot of people get notified when you make PRs, and this is a big waste of people's time.

File: files/en-us/web/css/css_backgrounds_and_borders/box-shadow_generator/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+testing---
```
[the remark this answers is not in the document]
Comment (jeeva11111) [reply files/en-us/web/css/css_backgrounds_and_borders/box-shadow_generator/index.md:1]:
```suggestion
testing---
```

**Review by jeeva11111 — COMMENTED** [review]
+local host---
title : Post request

**Comment by hamishwillee** [comment]
@jeeva11111 As you have [ignored the request above](https://github.com/mdn/content/pull/32252#issuecomment-1941602913) to not use this site as a sandbox you've been reported and will be blocked.

**Pull request closed without merging**